pkg/internalregistry/controllers/service_account_controller.go

```diff
@@ -68,12 +68,6 @@ func NewServiceAccountController(kubeclient kubernetes.Interface, serviceAccount
 				c.queue.Add(key)
 			}
 		},
-		DeleteFunc: func(obj any) {
-			key, err := cache.DeletionHandlingMetaNamespaceKeyFunc(obj)
-			if err == nil {
-				c.queue.Add(key)
-			}
-		},
```

Comment:
Nothing for this controller to do on deletion of a service account. Note that a deleted service account might still be queued by the Secret handler below.